crates/hc_sandbox/src/calls.rs

```diff
@@ -224,7 +224,9 @@ pub async fn call(holochain_path: &Path, req: Call) -> anyhow::Result<()> {
                 Ok(cmd) => cmds.push((cmd, None)),
                 Err(e) => {
                     if let holochain_websocket::WebsocketError::Io(e) = &e {
-                        if let std::io::ErrorKind::ConnectionRefused = e.kind() {
+                        if let std::io::ErrorKind::ConnectionRefused
+                        | std::io::ErrorKind::AddrNotAvailable = e.kind()
+                        {
```

Comment:
```suggestion
                        if let
                        #[cfg(not(target_os = "macos"))]
                        std::io::ErrorKind::ConnectionRefused
                        #[cfg(target_os = "macos")]
                        std::io::ErrorKind::AddrNotAvailable = e.kind()
                        {
```

Reply:
@freesig ping

Reply:
Is it not better to just cover both cases?

Reply:
my suggestion is based on the idea that if one of the errors happen on the platform we didn't expect them on the treatment might be different. if that's not the case then my suggestion can be ditched